orchestration/templates/create-dataset.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: create-dataset
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: dataset-name
+          - name: bq-project
+          - name: dataset-description
+          - name: dataset-expiration
+      script:
```

Comment:
I think this is simple enough that it could be a `container` instead of a `script`, but 100% ok leaving it as-is if it's working.